### ISSUE TITLE
Dimensions are not equal

### ISSUE THREAD

**Issue by chezhqant** [zjjMaiMai/Deep-Alignment-Network-A-convolutional-neural-network-for-robust-face-alignment#21]
def __gen_heatmap(self,shapes):
        shapes = shapes[:,:,tf.newaxis,tf.newaxis,:]
        value = self.__pixels__ - shapes
        value = tf.norm(value,axis=-1)
        value = 1.0 / (tf.reduce_min(value,axis=1) + 1.0)
        value = tf.expand_dims(value,axis=-1)
        return value

self.__pixels__ 和shapes的维度不一样呀？
